integration/tests/managed/managed_nodegroup_test.go

```diff
@@ -425,37 +423,59 @@ var _ = Describe("(Integration) Create Managed Nodegroups", func() {
 
 		Context("and creating a nodegroup with an update config", func() {
 			PIt("defining the UpdateConfig field in the cluster config", func() {
+				By("creating it")
 				updateConfig := &api.NodeGroupUpdateConfig{
 					MaxUnavailable: aws.Int(2),
 				}
-				clusterConfig := api.NewClusterConfig()
-				clusterConfig.Metadata.Name = params.ClusterName
-				clusterConfig.Metadata.Region = params.Region
-				clusterConfig.Metadata.Version = params.Version
+				clusterConfig := makeClusterConfig()
 				clusterConfig.ManagedNodeGroups = []*api.ManagedNodeGroup{
 					{
 						NodeGroupBase: &api.NodeGroupBase{
-							Name: "updateConfig",
+							Name: "update-config-ng",
 						},
 						UpdateConfig: updateConfig,
 					},
 				}
-
-				data, err := json.Marshal(clusterConfig)
-				Expect(err).ToNot(HaveOccurred())
-
 				cmd := params.EksctlCreateCmd.
 					WithArgs(
 						"nodegroup",
 						"--config-file", "-",
 						"--verbose", "4",
 					).
 					WithoutArg("--region", params.Region).
-					WithStdin(bytes.NewReader(data))
+					WithStdin(testutils.ClusterConfigReader(clusterConfig))
+
 				Expect(cmd).To(RunSuccessfully())
 			})
 		})
 
+		Context("and updating a nodegroup", func() {
+			It("should update a nodegroup", func() {
+				clusterConfig := makeClusterConfig()
+				clusterConfig.ManagedNodeGroups = []*api.ManagedNodeGroup{
+					{
+						NodeGroupBase: &api.NodeGroupBase{
+							Name: initialNodeGroup,
+						},
+						Spot: false,
+					},
+				}
+				cmd := params.EksctlUpdateCmd.
+					WithArgs(
+						"nodegroup",
+						"--config-file", "-",
+						"--verbose", "4",
+					).
+					WithoutArg("--region", params.Region).
+					WithStdin(testutils.ClusterConfigReader(clusterConfig))
+
+				Expect(cmd).To(RunSuccessfullyWithOutputStringLines(
+					ContainElement(ContainSubstring("unchanged fields")),
+					ContainElement(ContainSubstring("Spot")),
```

Comment:
whats this testing?

Reply:
I believe it is testing that other config is ignored.

I know we don't normally do this and each command only looks at what is cares about, but since this is `update` and we often get asked things like "Can i update xyz on my ng without recreating?" it may make sense to be very clear and log the things in the config which are not going to be changed.

Reply:
yes^ 👏 

I'd say informing the user of which fields were _not_ affected is so important for clarity of use that it's almost a feature and should be tested as such... it's part of making this command extra crystal clear to compensate for having a potentially confusing UX (`update` vs `upgrade`, `updateConfig` for upgrades)

Reply:
I'd (probably) be confused 😛